App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/ManageJob/JobSummaryCard.jsx

```diff
@@ -72,54 +128,11 @@ export class JobSummaryCard extends React.Component {
                         </div>
                     </CardContent>
                 </Card>
-            );
-
-            this.state.cards.push(card);
-        });
-    }
-
-    componentDidMount() {
-        this.init();
-    };
-
-    selectJob(id) {
-        var cookies = Cookies.get('talentAuthToken');
-        //url: 'https://talentservicestalentanderson.azurewebsites.net/listing/listing/GetJobByToEdit',
-    }
-
-    closeJob(id) {
-        var url = 'https://talentservicestalentanderson.azurewebsites.net/listing/listing/closeJob';
-        var cookies = Cookies.get('talentAuthToken');
-
-        $.ajax({
-            url: url,
-            headers: {
-                'Authorization': 'Bearer ' + cookies,
-                'Content-Type': 'application/json'
-            },
-            type: "POST",
-            contentType: "application/json",
-            data: JSON.stringify(id),
-            dataType: "json",
-            success: function (res) {
-                if (res.success == true) {
-                    TalentUtil.notification.show(res.message, "success", null, null);
-                    window.location = "/ManageJobs";
-                   
-                } else {
-                    TalentUtil.notification.show(res.message, "error", null, null)
-                }
-            }.bind(this),
-            error: function (res) {
-                console.error(res.status)
-            }
-        })
-    }
+        ));
 
-    render() {
         return (
             <CardGroup itemsPerRow={2}>
-                {this.state.cards}
+                {cards}
```

Comment:
When rendering certain element or data, it's better if we can have a proper naming convention so you and other developers can be able to determine easily which are functions and which are states. For example, this **card** might be mistakenly identify as a state. You can change this into something like **renderCards** so it is easier to understad.